### ISSUE TITLE
[Typescript] the declaration file doesnt work right if your module format is commonjs

### ISSUE THREAD

**Issue by aarondail** [rebeccahughes/react-native-device-info#208]
This is kinda a weird issue, that I don't fully understand, but if your tsconfig includes:
```
  "compilerOptions": {
    "module": "commonjs",
```

Then when you try to use this library in typescript:
```
import ReactNativeDeviceInfo from 'react-native-device-info';
console.log(ReactNativeDeviceInfo.getUniqueID());
```
You will get an error at runtime saying something like `react_native_device_info1.default` is undefined.

If you change your module to es2015 then it goes away.

I don't fully understand the problem, but I believe this is because the typescript declaration file currently "default" exports a class but looking at the actual sourcecode the `module.exports` is not exporting anything with the special name "default".  So there is a mismatch I think between the declaration file and the actual code.

Instead of forcing a change to the module type, Id like to propose a fix to the declaration file included in the package.  If I change the .d.ts file to:
```
  export function getUniqueID(): string;
  export function getManufacturer(): string;
  export function getBrand(): string;
  export function getModel(): string;
  export function getDeviceId(): string;
  export function getSystemName(): string;
  export function getSystemVersion(): string;
  export function getBundleId(): string;
  export function getBuildNumber(): string;
  export function getVersion(): string;
  export function getReadableVersion(): string;
  export function getDeviceName(): string;
  export function getUserAgent(): string;
  export function getDeviceLocale(): string;
  export function getDeviceCountry(): string;
  export function getTimezone(): string;
  export function getInstanceID(): string;
  export function isEmulator(): boolean;
  export function isTablet(): boolean;
  export function isPinOrFingerprintSet(cb: (isSet: boolean) => void): void;
```

And then change my import statement to:
```
import * as ReactNativeDeviceInfo from 'react-native-device-info';
```

Everything is ok.

**Comment by itchingpixels** [comment]
Yep, this pull request: https://github.com/rebeccahughes/react-native-device-info/commit/12c93cbe36af2489d5710f0d300ae4c08622cf7c has broken the TS type definitions for us too (using commonjs). Will open a PR to revert it.

**Comment by machour** [comment]
This problem have been fixed in #221, thank you @itchingpixels !
